core/controllers/feedback.py

```diff
@@ -233,8 +232,8 @@ class FeedbackThreadViewEventHandler(base.BaseHandler):
     """
 
     @acl_decorators.can_comment_on_feedback_thread
-    def post(self, exploration_id):
-        thread_id = self.payload.get('thread_id')
+    def post(self, thread_id):
+        exploration_id = thread_id.split('.')[0]
```

Comment:
Does this need a TODO to generalize later (to threads that aren't exploration-related)?

Reply:
Added a get_exp_id_from_thread_id function in feedback_domain. This function will need to be edited while generalizing.